src/global.scss

```diff
@@ -9,6 +9,14 @@
 @import '~@ionic/angular/css/text-alignment.css';
 @import '~@ionic/angular/css/text-transformation.css';
 @import '~@ionic/angular/css/flex-utils.css';
+@import '~quill/dist/quill.core.css';
+@import '~quill/dist/quill.bubble.css';
+@import '~quill/dist/quill.snow.css';
+
+[contenteditable] {
+  -webkit-user-select: text;
```

Comment:
What is the purpose of this?
Is it for drag-and-drop of images?
Please add comment if possible :)

Reply:
Required for QuillEditor. Need to discover why.